simple_ldap_user/simple_ldap_user.module

```diff
@@ -626,3 +628,37 @@ function simple_ldap_user_variable_get($name, $default = NULL, $force_default =
 
   return variable_get($name, $default);
 }
+
+/**
+ * Return the user account control status under Active Directory in LDAP database
+ * Check whether the user is disabled (Active Directory only).
+ * http://support.microsoft.com/kb/305144
+ *
+ * parameter: $ldap_user
```

Comment:
Please use PHPDoc formatting for the `@param` and `@return`.
https://docs.phpdoc.org/references/phpdoc/index.html